plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -9010,6 +9010,56 @@ public void testCollidingMixedCaseProperty()
         assertUpdate("DROP TABLE %s".formatted(tableName));
     }
 
+    @Test
+    public void testCommentViaSeparateCommentOnColumnStatement()
+    {
+        String table = "test_comment_on_" + randomNameSuffix();
+
+        assertUpdate("""
+                CREATE TABLE hive.tpch.%s (
+                   regular_column date,
+                   partition_column date
+                )
+                WITH (
+                   format = 'PARQUET',
+                   partitioned_by = ARRAY['partition_column']
+                )
+                """.formatted(table));
+
+        assertUpdate("COMMENT ON COLUMN %s.regular_column IS 'regular column comment'".formatted(table));
+        assertThat(getColumnComment(table, "regular_column")).isEqualTo("regular column comment");
+        assertUpdate("COMMENT ON COLUMN %s.partition_column IS 'partition column comment'".formatted(table));
+        assertThat(getColumnComment(table, "partition_column")).isEqualTo("partition column comment");
+        assertUpdate("COMMENT ON TABLE %s IS 'table comment'".formatted(table));
+        assertThat(getTableComment("hive", "tpch", table)).isEqualTo("table comment");
+
+        assertUpdate("DROP TABLE " + table);
+    }
+
+    @Test
+    public void testCommentViaCreateTableStatement()
+    {
+        String table = "test_comment_on_" + randomNameSuffix();
```

Comment:
This test doesn't run `COMMENT ON` statement. I would change to `test_create_table_partition_column_comment_` or something.